src/render_graph.rs

```diff
@@ -159,6 +159,19 @@ impl Node for PointCloudNode {
             tracked_pass.set_camera_viewport(viewport);
         }
         tracked_pass.set_render_pipeline(eye_dome_pipeline);
+
+        let edl_strength: f32 = if view.projection.z_axis.w == -1.0 {
+            // perspective projection
+            1.0
+        } else {
+            // orthographic projection
+            let dist = 1.0 / view.projection.z_axis.z; // far - near
```

Comment:
Can you omit the variable and leave a comment mentioning that this calculation depends on the code in `OrthographicProjection::get_projection_matrix`?

Reply:
Yep i've included a reference to glam's math on the orthographic rh projection matrix.